src/mpl/include/mpl_trmem.h

```diff
@@ -230,6 +230,89 @@ typedef struct {
   Utility
   M*/
 
+#ifdef HAVE_MM256_STREAM_SI256
+#include <immintrin.h>
+#endif
+#include "string.h"
+
+static inline void MPL_Memcpy_stream(void *dest, const void *src, size_t n)
+{
+#ifdef HAVE_MM256_STREAM_SI256
+    if (n <= 256) {
```

Comment:
For example, `n` is in bytes, but `256` meant for `bits` I assume? -- (since you replied, I guess my assumption was incorrect?)

Reply:
Ah. I see the confusion now. There's definitely some mixing of terminology here. Here at the top, this is using a regular `memcpy` if there are fewer than 256 bytes because below, we unroll the streaming copies to do 256 *bytes* per loop (in 256 *bit* chunks at a time). I'll improve the comments around this to make it more clear.